examples/ancestral-structure/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
```

Comment:
It might be helpful to have a small synopsis describing what this example does.

Reply:
This was an example of interfacing with PV using a smaller tree to get started, that eventually merged with the larger tree. Will delete